test/e2e/common/util.go

```diff
@@ -134,10 +134,11 @@ func NewSVCByName(c clientset.Interface, ns, name string) error {
 }
 
 // NewRCByName creates a replication controller with a selector by name of name.
-func NewRCByName(c clientset.Interface, ns, name string, replicas int32, gracePeriod *int64) (*v1.ReplicationController, error) {
+func NewRCByName(c clientset.Interface, ns, name string, replicas int32, gracePeriod *int64, args []string) (*v1.ReplicationController, error) {
```

Comment:
Thoughts on `containerArgs` or `cmdArgs` instead? It's not clear right away to me this is the args for the container spec.

Reply:
Good point, I like `containerArgs` since that's (ultimately) what it's passed to.